modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -46,6 +48,7 @@ namespace detail
         CV_RECT,       // cv::Rect user G-API data
         CV_SCALAR,     // cv::Scalar user G-API data
         CV_MAT,        // cv::Mat user G-API data
+        CV_GMAT,       // cv::GMat G-API data
```

Comment:
There's no data in GMat...

`GArray<GMat>` expands to a `vector<cv::Mat>` in the runtime, so a `CV_MAT` path should be followed in this case.

Reply:
Right, removed